tableau_utilities/scripts/cli.py

```diff
@@ -200,6 +202,14 @@
                                       'Use with --merge_with generate_merge_all')
 parser_config_merge.set_defaults(func=merge_configs)
 
+# APPLY CONFIGS
+parser_config_apply = subparsers.add_parser(
+    'apply_configs', help='Applies a config to a datasource. Writes over any datasource attributes to make it '
+                         'conform to the config.', formatter_class=RawTextHelpFormatter)
+parser_config_apply.add_argument('-cc', '--column_config', help='The path to the column configs file')
+parser_config_apply.add_argument('-cac', '--calculated_column_config', help='The path to the calculated field config file.')
```

Comment:
```suggestion
parser_config_apply.add_argument('-ccc', '--calculated_column_config', help='The path to the calculated field config file.')
```
- Just a nit-pick to match the abrehviation
- Also, maybe could even use `-Cc` if you wanna get fancy and keep it at 2 characters per command 😆

Reply:
I don't like making it case sensitive but I'll go with the 3 c's for now!